src/plugins/intel_npu/src/plugin/include/model_version.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "npu.hpp"
+
+namespace intel_npu {
+
+struct MetadataVersion {
+    uint32_t major;
+    uint32_t minor;
+} typedef MetadataVersion;
+
+struct OpenvinoVersion {
+    const std::string version;
+} typedef OpenvinoVersion;
+
+struct ModelLayout {
+    int something;
+    double somethingElse;
```

Comment:
Not sure if this kind of variable naming is allowed for production code, can we find something more technically?